src/interfaces/common.ts

```diff
@@ -9,6 +9,27 @@ export enum BooleanEnum {
 	false = "false",
 }
 
+// All variables below were defined by DataPortal. Do not change their values as it will affect our metrics logs and dashboards.
+export enum AnalyticsConfigEnum {
+	Product = "gitHubForJira", // required - do not change - https://hello.atlassian.net/browse/DE-8853
+	ProdEnv = "prod"
+}
```

Comment:
If the product is a constant, don't make it an enum.  That doesn't really make any sense.  It's only used when creating the client, just use it inline.

Also, `ProdEnv` is not correct here. It should just be `env` as the analytics could be be used for prod or staging or ddev.